plugins/helm/helm.plugin.zsh

```diff
@@ -14,9 +14,9 @@ command mkdir -p "$ZSH_CACHE_DIR/completions"
 # If the completion file does not exist, generate it and then source it
 # Otherwise, source it and regenerate in the background
 if [[ ! -f "$ZSH_CACHE_DIR/completions/_helm" ]]; then
-  helm completion zsh >| "$ZSH_CACHE_DIR/completions/_helm"
+  helm completion zsh | tee "$ZSH_CACHE_DIR/completions/_helm"
```

Comment:
The output of `tee` should be silenced, otherwise it'll be shown on the terminal as well: `tee <file> >/dev/null`